src/page/address-step.page.ts

```diff
@@ -4,7 +4,7 @@ export class AddressStepPage {
   private proceedToCheckout: ElementFinder;
 
   constructor () {
-    this.proceedToCheckout = $('#center_column > form > p > button > span');
+    this.proceedToCheckout = $('form .cart_navigation > button[type="submit"]');
```

Comment:
could you try to use a different attribute than type? for example, name attribute, will not be susceptible to changes and give us more context about what we are looking for

Reply:
Done